### DISCUSSION TITLE
Static (minified) .js file for octokit rest

### DISCUSSION THREAD

**Discussion by zacun** [octokit/octokit.js#2049] [Q&A] [answered]
Hello,

I'm working on a project with ThunderBird and GitHub API, as such I chose to work with Octokit.
However, because of some CSP issues I can't use the `import ... from cdn...` so I'd like to get a static min.js file with all the code if possible (just like for jQuery for instance)

I read your discussion here : https://github.com/octokit/rest.js/discussions/1745 and tried to download the `source code.zip` at https://github.com/octokit/rest.js/releases/tag/v17.9.2 which you gave as an answer but I could not find any `xxx.min.js` file inside it.

Did I miss something ?

Thank you for your time.

Best Regards,

**Comment by wolfy1339** [comment, marked as answer]
You could save the code from the CDN.
I don't think we publish any artifacts on GitHub releases, they get pushed to npmi

**Reply by gr2m** [reply]
> You could save the code from the CDN.

unfortunately the code from the CDN, at least from skypack, is still using imports

We don't provide a built version any more, you'll have to build it yourself using webpack or similar

**Reply by zacun** [reply]
I see, thank you for your answer.
